docs/renarde/README.md

```diff
@@ -0,0 +1,7 @@
+# Renarde support
+
+[Renarde](https://quarkiverse.github.io/quarkiverse-docs/quarkus-renarde/dev/index.html) is a server-side Web Framework based on Quarkus, Qute, Hibernate and RESTEasy Reactive.
+
+[vscode-quarkus](https://github.com/redhat-developer/vscode-quarkus) provides support for writing `Renarde` applications in:
+
+ * [Java files](JavaSupport.md#java-support) : handle Renarde's custom logic for how REST endpoints are determined in `Controller` classes.
```

Comment:
in [Controller](https://quarkiverse.github.io/quarkiverse-docs/quarkus-renarde/dev/index.html#_controllers) classes.